### DISCUSSION TITLE
Did you planned to tag ?

### DISCUSSION THREAD

**Discussion by rramillien** [nothings/stb#1240] [Q&A] [answered]
Hello,

You're classes seems awesome, but I would like to use them using cmake and FetchContent.

However, if I fetch content, I can only do this on a branch and not on a tag or release because your project don't have one.
It means that because I do not control the version of your code in my project, it can break the compilation if you made an update on your branch.

So, I'd like to know if you planned to make some tags or release or pre-releases or if the only solution I have is to copy-paste the files I need into my own project ?

Best regards,

Régis Ramillien

**Comment by mmozeiko** [comment, marked as answer]
GIT_TAG in FetchContent accepts tag, branch and also **commit id**. So you can choose whichever commit you want from [commit list](https://github.com/nothings/stb/commits/master), like currently latest af1a5bc, and put that in your cmake file.

**Reply by rramillien** [reply]
Hello,

Great, I didn't know. I will do this.

Regards,

Régis
